modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -963,6 +996,26 @@ private void _addRecurrenceOptions(
 		}
 	}
 
+	private void _deletePowwowMeetingOccurrence(long powwowMeetingId) {
+
+		PowwowMeeting powwowMeeting =
```

Comment:
should we collect this logic into a method `PowwowMeetingOccurrenceLocalServiceUtil.deleteByPowwowMeetingId`. In the method, find all occurrence and remove it there?